package.json

```diff
@@ -115,6 +115,6 @@
         "eslint-plugin-security": "^1.4.0"
     },
     "engines": {
-        "node": ">=14"
+        "node": ">=16"
```

Comment:
Out of curiosity: Are changes in `package-lock.json` related to Node.js version update?

Reply:
They are related to npm upgrade

Reply:
OK 👍🏻